### PR TITLE
upgrade recipe for patchlevel/event-sourcing 3.6

### PR DESCRIPTION
**Pull request by DavidBadura**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/patchlevel/event-sourcing

### CODE REVIEW

File: patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore
Comment (diimpp) [inline patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore:1]: `migrations` dir is part of doctrine recipe, so I don't think this package's recipe should be adding it.

https://github.com/symfony/recipes/tree/main/doctrine/doctrine-migrations-bundle/3.1/migrations

File: patchlevel/event-sourcing-bundle/3.6/manifest.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                "    POSTGRES_DB: ${POSTGRES_DB:-app}",
+                "    # You should definitely change the password in production",
+                "    POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-!ChangeMe!}",
+                "    POSTGRES_USER: ${POSTGRES_USER:-app}",
+                "  volumes:",
+                "    - database_data:/var/lib/postgresql/data:rw",
+                "    # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!",
+                "    # - ./docker/db/data:/var/lib/postgresql/data:rw"
+            ],
+            "volumes": ["eventstore_data:"]
```
Comment (diimpp) [inline patchlevel/event-sourcing-bundle/3.6/manifest.json:34]: Not part of my recipe review, but this volume don't appear to be in use.

File: patchlevel/event-sourcing-bundle/3.6/manifest.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+    "bundles": {
+        "Patchlevel\\EventSourcingBundle\\PatchlevelEventSourcingBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "migrations/": "migrations/"
```
Comment (diimpp) [inline patchlevel/event-sourcing-bundle/3.6/manifest.json:7]:
```suggestion
        "config/": "%CONFIG_DIR%/"
```

File: patchlevel/event-sourcing-bundle/3.6/manifest.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                "    # You may use a bind-mounted host directory instead, so that it is harder to accidentally remove the volume and lose all your data!",
+                "    # - ./docker/db/data:/var/lib/postgresql/data:rw"
+            ],
+            "volumes": ["eventstore_data:"]
+        },
+        "docker-compose.override.yml": {
+            "services": [
+                "eventstore:",
+                "  ports:",
+                "    - \"5432\""
```
Comment (diimpp) [inline patchlevel/event-sourcing-bundle/3.6/manifest.json:40]: Not part of my recipe review, what's the point in splitting docker configuration into two directives?

**Review by diimpp — CHANGES_REQUESTED** [review]
`Request changes` is due `migrations` dir.

File: patchlevel/event-sourcing-bundle/3.6/manifest.json (hunk 3 above)
Comment (DavidBadura) [reply patchlevel/event-sourcing-bundle/3.6/manifest.json:40]: This is for the compose.override.yaml

File: patchlevel/event-sourcing-bundle/3.6/manifest.json (hunk 1 above)
Comment (DavidBadura) [reply patchlevel/event-sourcing-bundle/3.6/manifest.json:34]: good catch

File: patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore
Comment (diimpp) [reply patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore:1]: Ah, I think I understand now -- your package depends on the `doctrine/migrations` and not `doctrine/migrations-bundle`, hence you don't have symfony/recipe for `doctrine/migrations-bundle` executed.

**Review by diimpp — APPROVED** [review]

File: patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore
Comment (DavidBadura) [reply patchlevel/event-sourcing-bundle/3.6/migrations/.gitignore:1]: Okay, to clarify a bit more: We are based on doctrine libs, but not on doctrine-bundle. The doctrine-bundle is optional. If you only want to use event sourcing, then you don't need to require the doctrine-bundle.

In this case, unfortunately, the folder does not exist

**Pull request merged**